cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -313,8 +313,19 @@ public void setContainerEnvVars(List<ContainerEnvVar> envVars) {
         templateContainerEnvVars = envVars;
     }
 
-
     public void setContainerSecurityContext(SecurityContext securityContext) {
         templateContainerSecurityContext = securityContext;
     }
+
+    /**
+     * Transforms properties to log4j2 properties file format and adds property for reloading the config
+     * @param properties map with properties
+     * @return modified string with monitorInterval
+     */
+    @Override
+    public String createPropertiesString(OrderedProperties properties) {
```

Comment:
The javadoc suggests that this is Log4j specific method? If yes, it should be reflected in the name ... e.g. `createLog4j2Properties`

Reply:
It is called from the abstract model class (https://github.com/strimzi/strimzi-kafka-operator/blob/8f49fffc53fa2f0464e4fa4009fd1c644b02bb42/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java#L386) 
I am not sure whether renaming this is worth of overriding entire `parseLogging` method.

Reply:
If that function is specific to log4j you should name it like that. I'm not sure you need to override anything - just rename it eevrywhere, or? Ad if it is not specific for Log4j, it should not say so in the comment.